oauth2client/contrib/appengine.py

```diff
@@ -89,11 +89,11 @@ def _safe_html(s):
     return cgi.escape(s, quote=1).replace("'", '&#39;')
 
 
-class InvalidClientSecretsError(Exception):
-    """The client_secrets.json file is malformed or missing required fields."""
+class Error(Exception):
+    """Base error for this module."""
 
 
-class InvalidXsrfTokenError(Exception):
+class InvalidXsrfTokenError(Error):
```

Comment:
I'm thinking we may want to keep it, although I'm not the expert on it.  How else would you want to handle invalid XSRF tokens?